containers/osbuild-composer/entrypoint.py

```diff
@@ -194,12 +215,12 @@ def _prepare_sockets(self):
 
         # osbuild-remote-worker.socket
         if self.args.remote_worker_api:
-            print("Create remote-worker-api socket", file=sys.stderr)
+            print("Create remote-worker-api socket on address {}:{}".format(self.args.remote_worker_api_bind_address, self.args.remote_worker_api_port), file=sys.stderr)
```

Comment:
We prefer f-strings in osbuild project, can you please adjust it?

Reply:
Also, can you put the address in brackers since it's an IPv6 one? https://en.wikipedia.org/wiki/IPv6_address#Literal_IPv6_addresses_in_network_resource_identifiers

Reply:
Done and Done